src/policy/marksweepspace/native_ms/global.rs

```diff
@@ -78,6 +79,9 @@ pub struct MarkSweepSpace<VM: VMBinding> {
     /// these block lists in the space. These lists are only filled in the release phase,
     /// and will be moved to the abandoned lists above at the end of a GC.
     abandoned_in_gc: Mutex<AbandonedBlockLists>,
+    /// Count the number of pending release packets during the `Release` stage.
```

Comment:
The comments seem too general. There might be other work packets that the plan is not aware of. The plan only cares about packets that mutate block lists, not all the packets in the release stage.

Reply:
Yes. That makes sense.  I'll explicitly mention `ReleaseMarkSweepSpace` and `ReleaseMutator` in the comment.